docs/source/examples/notebooks/initialize-model-with-solution.ipynb

```diff
@@ -89,7 +89,9 @@
     "param = model.default_parameter_values\n",
     "current_interpolant = pybamm.Interpolant(drive_cycle[:, 0], drive_cycle[:, 1], pybamm.t)\n",
     "# set drive cycle\n",
-    "param[\"Current function [A]\"] = current_interpolant"
+    "param[\"Current function [A]\"] = current_interpolant\n",
+    "# set upper voltage cut-off high to avoid the cutoff\n",
+    "param[\"Upper voltage cut-off [V]\"] = 4.3"
```

Comment:
Is this a symptom of some other issue? Seems weird to have to change the parameter set

Reply:
I modified the example and removed the line changing the upper voltage cutoff